resolwe/flow/tests/test_templatetags.py

```diff
@@ -0,0 +1,70 @@
+# pylint: disable=missing-docstring
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
+import shutil
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from resolwe.flow.managers import manager
+from resolwe.flow.models import Process, Data
+
+
+class ProcessFieldsTagsTest(TestCase):
```

Comment:
A long test for such small functionality. Couldn't you just create a Data obj and call Django's render function? What about border cases, e.g. no data object?